src/tests/ftest/network/cart_self_test.py

```diff
@@ -104,21 +100,30 @@ def setUp(self):
         # Start the daos server
         self.start_server_managers()
 
+        # Generate a uri file using daos_agent dump-attachinfo
+        attachinfo_file = "{}.attach_info_tmp".format(self.server_group)
+        self.uri_file = os.path.join(self.tmp, attachinfo_file)
+        agent_cmd = self.agent_managers[0].daos_agent
+        agent_cmd.dump_attachinfo(self.uri_file)
+
+
     def test_self_test(self):
         """Run a few CaRT self-test scenarios.
 
-        :avocado: tags=all,smoke,unittest,tiny,cartselftest
+        :avocado: tags=all,pr,smoke,unittest,tiny,cartselftest
         """
         # Setup the orterun command
-        orterun = Orterun(SelfTest(self.cart_bin))
-        orterun.ompi_server.update(
-            "file:{}".format(self.uri_file), "orterun/ompi_server")
+        self_test = SelfTest(os.path.join(self.prefix, "bin"))
+        orterun = Orterun(self_test)
         orterun.map_by.update(None, "orterun/map_by")
         orterun.enable_recovery.update(False, "orterun/enable_recovery")
 
         # Get the self_test command line parameters
         orterun.job.get_params(self)
-        orterun.job.group_name.value = self.server_group
+        orterun.job.group_name.update(self.server_group)
+        orterun.job.message_sizes.update(
+            self.params.get("size", "/run/muxtestparams/message_size/*")[0])
+        orterun.job.attach_info.update(self.tmp)
```

Comment:
It may be safer to base this on the `self.uri_file` definition - which will be needed if my previous comment results in a change to it's definition:
```suggestion
        orterun.job.attach_info.update(os.path.dirname(self.uri_file), "attach_info")
```

Also the `update()` method has an optional 2nd argument (the name of the property), which if used will log the variable assignment - which can be useful for triaging failures.

Reply:
Done.